lib/util/noop

```diff
@@ -0,0 +1 @@
+module.exports = function() {}
```

Comment:
Is there any reason not to inline this?

Reply:
I broke this out as a separate module so that it could be used whereever a noop function is required, which is currently in 2 places in the library... but didn't then update the second usage (in the 'stupidPjax' fallback) 😊

Reply:
Can you rename the file to `noop.js`?

Reply:
Yes, nice catch, this is why code reviews are important 😉